firebase-perf/src/main/java/com/google/firebase/perf/metrics/validator/PerfMetricValidator.java

```diff
@@ -28,6 +28,8 @@
 /** An abstract class providing an interface to validate PerfMetric */
 public abstract class PerfMetricValidator {
 
+  private String format;
```

Comment:
Is this used?

Reply:
This is not used! Thanks for calling this out. Removed it.